integration-tests/oidc-wiremock/src/test/java/io/quarkus/it/keycloak/KeycloakTestResource.java

```diff
@@ -0,0 +1,99 @@
+
+import java.util.Collections;
+import java.util.Map;
+
+import javax.ws.rs.core.MediaType;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.WireMock;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+
+public class KeycloakTestResource implements QuarkusTestResourceLifecycleManager {
+
+    private WireMockServer server;
+
+    @Override
+    public Map<String, String> start() {
+
+        server = new WireMockServer(options().port(8180));
+        server.start();
+
+        WireMock.configureFor(8180);
+        WireMock.stubFor(
+                get(urlEqualTo("/auth/realms/quarkus/.well-known/openid-configuration"))
+                        .willReturn(aResponse()
+                                .withHeader("Content-Type", MediaType.APPLICATION_JSON)
+                                .withBody("{\n" +
+                                        "    \"authorization_endpoint\": \"http://localhost:8180/authenticate\",\n" +
+                                        "    \"end_session_endpoint\": \"http://localhost:8180/logout\",\n" +
+                                        "    \"id_token_signing_alg_values_supported\": [\n" +
+                                        "        \"RS256\",\n" +
+                                        "        \"ES256\",\n" +
+                                        "        \"HS256\"\n" +
+                                        "    ],\n" +
+                                        "    \"issuer\": \"http://localhost:8180/auth/realms/quarkus\",\n" +
+                                        "    \"jwks_uri\": \"http://localhost:8180/auth/realms/quarkus/protocol/openid-connect/certs\",\n"
+                                        +
+                                        "    \"response_types_supported\": [\n" +
+                                        "        \"code\",\n" +
+                                        "        \"code id_token\",\n" +
+                                        "        \"id_token\",\n" +
+                                        "        \"token id_token\"\n" +
+                                        "    ],\n" +
+                                        "    \"subject_types_supported\": [\n" +
+                                        "        \"public\"\n" +
+                                        "    ],\n" +
+                                        "    \"token_endpoint\": \"http://localhost:8180/auth/realms/quarkus/protocol/openid-connect/token\"\n"
+                                        +
+                                        "}")));
+
+        WireMock.stubFor(
+                get(urlEqualTo("/auth/realms/quarkus/protocol/openid-connect/certs"))
+                        .willReturn(aResponse()
+                                .withHeader("Content-Type", MediaType.APPLICATION_JSON)
+                                .withBody("{\n" +
+                                        "\"keys\": [\n" +
```

Comment:
@cemnura please return the content of this file here,
https://github.com/smallrye/smallrye-jwt/blob/master/testsuite/basic/src/test/resources/encryptPublicKey.jwk (rename it to `publicKey.jwk`) - and please add a `kid` property, say, `1`